superset-frontend/spec/javascripts/dashboard/components/PropertiesModal_spec.jsx

```diff
@@ -59,7 +59,8 @@ fetchMock.get('glob:*/api/v1/dashboard/*', {
   },
 });
 
-describe('PropertiesModal', () => {
+// all these tests need to be moved to dashboard/components/PropertiesModal/PropertiesModal.test.tsx
+describe.skip('PropertiesModal', () => {
```

Comment:
All these tests need to be refactored since this PR changes the component to be functional. I'll work on these in a separate PR to avoid holding this PR too long

Reply:
I hate those enzyme tests, great to see them go one by one